Libplanet.RocksDBStore/RocksDBStoreBitConverter.cs

```diff
@@ -0,0 +1,53 @@
+using System.Text;
+
+namespace Libplanet.RocksDBStore
+{
+    /// <summary>
+    /// Collection of wrapper methods to convert to byte-arrays from various types to store
+    /// <see cref="RocksDBStore"/>.  Every integer data is converted to Big-endian, or
+    /// "Network Byte Order" to order index lexicographically.
+    /// </summary>
+    internal static class RocksDBStoreBitConverter
+    {
+        /// <summary>
+        /// Get <c>long</c> representation of the <paramref name="value"/>.
+        /// </summary>
+        /// <param name="value">The Big-endian byte-array value to convert to <c>long</c>.</param>
+        /// <returns>The <c>long</c> representation of the <paramref name="value"/>.</returns>
+        public static long ToInt64(byte[] value)
+        {
+            return NetMQ.NetworkOrderBitsConverter.ToInt64(value);
```

Comment:
As we've decided to have this our own utility class, it would be better if it's independent from NetMQ.  You've once implemented this without NetMQ already, right?

Reply:
You're right, I'll fix it.

Reply:
Changed implementation and amended the commit. https://github.com/planetarium/libplanet/pull/808/commits/7efba51a500a0e9a241136d149d6b4b701b6ac02